flink-end-to-end-tests/run-nightly-tests.sh

```diff
@@ -185,5 +198,32 @@ run_test "Local recovery and sticky scheduling end-to-end test" "$END_TO_END_DIR
 run_test "Local recovery and sticky scheduling end-to-end test" "$END_TO_END_DIR/test-scripts/test_local_recovery_and_scheduling.sh 4 10 rocks true true" "skip_check_exceptions"
 
 printf "\n[PASS] All bash e2e-tests passed\n"
+
+printf "\n\n==============================================================================\n"
+printf "Running Java end-to-end tests\n"
+printf "==============================================================================\n"
+
+set -x
+HERE="`dirname \"$0\"`"				# relative
```

Comment:
reduce whitespace

Reply:
ping

Reply:
are you talking about the vertical whitespace?

Reply:
yes

Reply:
Consider it gone :)